recipes/iptcinfo/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "IPTCInfo" %}
+{% set version = "1.9.5.post6" %}
```

Comment:
Please only include released versions

Reply:
Wow, their version scheme is not compatible with ours.  I definitley don't think you should add `post` because that has a different meaning in pip, etc. I would maybe replace the dash with a dot (since that is what it is semantically) or replace it with an underscore.

Reply:
@scopatz, thanks! I changed it to a .6 as you suggested. It should be ready to merge now.